CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
-
 ## 0.16.3 (unreleased)
 
+- experimental abstractions have been promoted to supported abstractions.  see new modules for:
```

Comment:
1. Can you capitalize the leading E here?
2. Since you are going to cut 0.16.3 today, can you please just put (Released January, 25th 2019) in the section title?

Reply:
sure.